src/hotspot/cpu/riscv/c2_MacroAssembler_riscv.hpp

```diff
@@ -230,4 +230,17 @@
     }
   }
 
+  void integer_extend_v(VectorRegister dst, BasicType dst_bt, int vector_length,
+                        VectorRegister src, BasicType src_bt);
+
+  void integer_narrow_v(VectorRegister dst, BasicType dst_bt, int vector_length,
+                        VectorRegister src, BasicType src_bt, VectorRegister tmp);
+
+  void vfcvt_rtz_xu_f_v_safe(VectorRegister dst, VectorRegister src);
```

Comment:
I don't think we need the unsigned version. Could you please remove them?

Reply:
Fixed.